src/main/webapp/player/simid_player.html

```diff
@@ -53,18 +52,59 @@
           break;
         case 'survey.html':
           let adParamsObj = [
-            {'question':'Would you like a survey?',
-             'answers':['Yes', 'No', 'Not Sure']},
-            {'question':'Are you certain?',
-             'answers':['Of course', 'Not Really', 'Well...']},
-            {'question':'What animals do you like?',
-             'answers':['Cats', 'Dogs', 'Other']}
+            {
```

Comment:
put it back to original formatting, takes up less space

Reply:
done